src/controller_adapters/base_gamepad_adapter.py

```
@@ -0,0 +1,61 @@
+from typing import List
```

Ruff (0.16.0)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)